### PR TITLE
Python: Add remaining labeled metric types

### PR DESCRIPTION
**Pull request by mdboom**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,9 @@
     `~/.gradle/glean`. Therefore `clean` can be used without reinstalling
     Miniconda afterward every time.
 * Python:
-  * The timing distribution and memory distribution metric types are now supported in Python ([#762](https://github.com/mozilla/glean/pull/762), [#763](https://github.com/mozilla/glean/pull/763)) 
+  * **Breaking Change**: The `glean.util` and `glean.hardware` modules, which
```
Comment (badboy) [inline CHANGELOG.md]: I assume this is coming from #764, and thus once that is merged this PR should apply cleanly?

File: docs/user/metrics/labeled_booleans.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Accessibility
+// Was anything recorded?
+assertTrue(Accessibility.features["screen_reader"].testHasValue())
+assertTrue(Accessibility.features["high_contrast"].testHasValue())
+// Do the booleans have the expected values?
+assertEquals(True, Accessibility.features["screen_reader"].testGetValue())
+assertEquals(False, Accessibility.features["high_contrast"].testGetValue())
+// Did we record any invalid labels?
+assertEquals(1, Accessibility.features.testGetNumRecordedErrors(ErrorType.InvalidLabel))
```
Comment (badboy) [inline docs/user/metrics/labeled_booleans.md]: Above example shouldn't record errors. Should we test equality against 0 here?

File: docs/user/metrics/labeled_booleans.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+```Swift
+@testable import Glean
+
+// Was anything recorded?
+XCTAssert(Accessibility.features["screen_reader"].testHasValue())
+XCTAssert(Accessibility.features["high_contrast"].testHasValue())
+// Do the booleans have the expected values?
+XCTAssertEqual(true, try Accessibility.features["screen_reader"].testGetValue())
+XCTAssertEqual(false, try Accessibility.features["high_contrast"].testGetValue())
```
Comment (badboy) [inline docs/user/metrics/labeled_booleans.md]: I wanted to complain that this doesn't have the `testGetNumRecordedErrors` line here, but I checked and we don't actually have that method on labeled metrics in Swift 🤔

We should file a bug.

File: docs/user/metrics/labeled_booleans.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+```Python
+# Was anything recorded?
+assert metrics.accessibility.features["screen_reader"].test_has_value()
+assert metrics.accessibility.features["high_contrast"].test_has_value()
+# Do the booleans have the expected values?
+assert metrics.accessibility.features["screen_reader"].test_get_value()
+assert not metrics.accessibility.features["high_contrast"].test_get_value()
+# Did we record any invalid labels?
+assert 1 == metrics.accessibility.features.test_get_num_recorded_errors(
```
Comment (badboy) [inline docs/user/metrics/labeled_booleans.md]: As above: `0 == ...`

File: docs/user/metrics/labeled_booleans.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+## Limits
+
+* Labels must conform to the [label formatting regular expression](index.md#label-format).
+
+* Labels support lowercase alphanumeric characters; they additionally allow for dots (`.`), underscores (`_`) and/or hyphens (`-`).
+
+* Each label must have a maximum of 60 bytes, when encoded as UTF-8.
+
+* If the labels are specified in the `metrics.yaml`, using a different label will be replaced with the special value `__other__`.
```
Comment (badboy) [inline docs/user/metrics/labeled_booleans.md]: I stumbled upon the "different".
Maybe "unknown" or "not listed" or something to make it clear that it's "different from the listed ones"

File: docs/user/metrics/labeled_booleans.md (hunk 3 above)
Comment (mdboom) [reply docs/user/metrics/labeled_booleans.md]: https://bugzilla.mozilla.org/show_bug.cgi?id=1621698

**Pull request merged**